code/game/mecha/mecha.dm

```diff
@@ -904,11 +906,12 @@
 			to_chat(user, "<span class='warning'>You can't enter the exosuit while buckled.</span>")
 		else if(user.has_buckled_mobs())
 			to_chat(user, "<span class='warning'>You can't enter the exosuit with other creatures attached to you!</span>")
+		else if(HAS_TRAIT(user,TRAIT_DUMB))
+			to_chat(user,"<span class='warning'>You can't drive dummy</span>")
 		else
 			moved_inside(user)
-	else
-		to_chat(user, "<span class='warning'>You stop entering the exosuit!</span>")
-	return
+			return
+		to_chat(user, "<span class='warning'>You stop entering the exosuit!</span>");

```

Comment:
You've removed a return here, so this line will _always_ be said, which I don't imagine is intended.

Reply:
The return is if the move into the mecha was successful. By this point, the only case left is failure of some type. It's inverted logic, but lolmechs